CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing to numpy
+
+## Reporting issues
+
+When reporting issues please include as much detail as possible about your
+operating system, numpy version and python version. Whenever possible, please
+also include a brief code example that demonstrates the problem.
```

Comment:
And add that for segfaults a Gdb traceback should be added, plus a link on how to do that (https://github.com/numpy/numpy/blob/master/doc/source/dev/development_environment.rst#debugging)